src/components/home/ProjectSection/Project.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Card from 'react-bootstrap/Card';
+import "./Project.css";
+
+type Props = {
+    name: string;
+    desc: string;
+    imageUrl: string;
+};
+
+class Project extends Component<Props>{
```

Comment:
Shall we use function-based component definition instead of class-based? I thought the function-based approach is more recommended now days.

Reply:
We can change them to function-based if that's preferred

Reply:
I posted a question in Slack to ask other people's opinion. I had the impression that funtional component was preferred for JS. I'm not sure about Typescript though. We can keep the code for now. Can refactor later if we decided to use functional component.

Reply:
Sure

Reply:
I saw the discussion in Slack and I agree it would be better to use functional components. Will make the change in the next pull request.